### PR TITLE
Mandla sprint 3 statistics popup

### PR DESCRIPTION
**Pull request by Khumalom966**

### CODE REVIEW

File: modules/scoreData.js [hunk 1]
```diff
@@ -1,12 +1,24 @@
 'use strict'
```
Comment (Mulowwits) [inline modules/scoreData.js:1]: looks good

File: public/scripts/indexScore.js [hunk 2]
```diff
@@ -1,34 +1,35 @@
 'use strict'
 
-const tries = [ 3, 0, 0, 0, 1, 0 ]
+let attemptsData = [ ]
```
Comment (Mulowwits) [inline public/scripts/indexScore.js:3]: good thing to remove static data

File: public/scripts/indexScore.js [hunk 3]
```diff
@@ -1,34 +1,35 @@
...
-    alert(e) 
+    else { throw 'Failed to load classlist: response code invalid!' }
+  })
+  .then(playerScore => {
+    playerScore.forEach(data => {
+      attemptsData = data.attempts
+    })
+    //attemptsData = playerScore.attempts.slice()
+  })
+  .catch(e => {
```
Comment (Mulowwits) [inline public/scripts/indexScore.js:27]: good practice to catch errors

**Review by Mulowwits — COMMENTED** [review]
Looks fine but normalize using comments on long code

File: public/scripts/indexScore.js (hunk 2 above)
Comment (Khumalom966) [reply public/scripts/indexScore.js:3]: In future I will make sure

**Pull request merged**